### PR TITLE
Adds new integration [basbruss/adaptive-cover]

### PR DESCRIPTION
**Pull request by basbruss**
## Checklist

- [x] I've read the [publishing documentation](https://hacs.xyz/docs/publish/start).
- [x] I've added the [HACS action](https://hacs.xyz/docs/publish/action) to my repository.
- [x] (For integrations only) I've added the [hassfest action](https://developers.home-assistant.io/blog/2020/04/16/hassfest/) to my repository.
- [x] The actions are passing without any disabled checks in my repository.
- [x] I've added a link to the action run on my repository below in the links section.
- [x] I've created a new release of the repository after the validation actions were run successfully.

## Links

Link to current release: <https://github.com/basbruss/adaptive-cover/releases/tag/v1.4.4>
Link to successful HACS action (without the `ignore` key): <https://github.com/basbruss/adaptive-cover/actions/runs/7694819607>
Link to successful hassfest action (if integration): <https://github.com/basbruss/adaptive-cover/actions/runs/7694819596>

### CODE REVIEW

**Comment by tjorim** [comment]
Be aware you do the 'HACS Validation' twice: once in the hacs.yaml, once in the validate.yml workflow. One of them is wrong (hint: the ignore key).

**Comment by basbruss** [comment]
> Be aware you do the 'HACS Validation' twice: once in the hacs.yaml, once in the validate.yml workflow. One of them is wrong (hint: the ignore key).

Thanks! I removed the one in validate.yml and renamed it to hassfest.yml.

File: integration [hunk 1]
```diff
@@ -96,6 +96,7 @@
   "bacco007/sensor.waternsw",
   "barban-dev/homeassistant-midea-dehumidifier",
   "barleybobs/homeassistant-ecowater-softener",
+  "basbruss/adaptive-cover",
```
Comment (ludeeus) [inline integration:99]: https://github.com/basbruss/adaptive-cover/actions/runs/11426903428/job/31790406035 hassfest is failing.

Comment (basbruss) [reply integration:99]: That's linked to this PR to core https://github.com/home-assistant/core/pull/128965

Comment (basbruss) [reply integration:99]: New release removes code that failed hassfest

**Pull request merged**